pkg/kubelet/dockertools/manager.go

```diff
@@ -1494,14 +1494,28 @@ func (dm *DockerManager) runContainerInPod(pod *api.Pod, container *api.Containe
 	if err != nil {
 		return kubecontainer.ContainerID{}, fmt.Errorf("InspectContainer: %v", err)
 	}
+
+	if !containerInfo.State.Running {
```

Comment:
The container can exit after this check too right? This check might catch a few more cases though..
The OOM score logic should be resilient to containers not existing. Is that not the case?